deconnable_everything.md

```diff
@@ -0,0 +1,107 @@
+<!-- It's not necessary to follow this format, as long as you provide a coherent and structured document -->
+
+## Abstract
+Dozens of snowflake and legacy objects are straight up indestructible in CM through normal means. Most people who play CM regulary grown used to it but it is 
+shockingly apparent when you actually to try to notice it or when someone abuses it. Things like filing cabinets, hydroponics trays ore boxes, storage tanks, showcases 
+and computers are indestructible through normal means (if at all) for god-knows-why and are constantly used for cheesing every now and then. If the CLF threw away their armor and
+just wore crates (100% bulletproof), they would control the galaxy already.
+
+Furthermore, many things also lack proper deconstruction. You should be able to deconstruct nearly everything destructible ( within balance bounds, of course)
+
+***Disclaimer: UNINTENTIONAL objects meaning things that aren't meant to be indestructable or affect balance is a big way if its destructable, like hulls ***
+<!-- An abstract is a short blurb, about a paragraph or two, succinctly describing your feature. This should mostly be "why", but can include "what". -->
```

Comment:
This should be a description of what most objects should be like, aka destructible. Doesn't need to be long or anything, just justify why they should be destructible. Don't refer to how things were in the past